src/Components/Hero/Hero.scss

```diff
@@ -1,6 +1,7 @@
-﻿$curiosityThemeColor: grey;
-$spiritThemeColor: grey;
-$opportunityThemeColor: grey;
+﻿//Please read README section on Styling before editing or creating SCSS
+
+@import "./src/ConstantsMixins";
```

Comment:
Does this work? 

I'd expect this to be looking for a file under `src/Components/Hero/src/ConstantsMixins` and then not find anything?

Reply:
seems to work!